e2e/prerender/playwright.config.js

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from '@playwright/test';
+import { config } from '../playwright-config';
+
+/** @type {import('@playwright/test').PlaywrightTestConfig} */
+export default {
+	...defineConfig(config),
+	webServer: {
+		command: 'pnpm sources --changed && pnpm build && pnpm preview',
```

Comment:
Is there a reason you needed to customize the behavior here?

`sources`, `build`, and `preview` are already included in the E2E action here:
https://github.com/evidence-dev/evidence/blob/812ea7506c4585366176beac39f5e2679de135a3/.github/workflows/e2e.yml#L68

The base playwright config also has `port`, `reuseExistingServer`, and `timeout` specified too
https://github.com/evidence-dev/evidence/blob/812ea7506c4585366176beac39f5e2679de135a3/e2e/playwright-config.js

---

If you made this change to only run `preview` mode tests (no `dev` mode tests), instead you should do something similar to what I did in `e2e/hmr`, which only runs tests in `dev` mode:
https://github.com/evidence-dev/evidence/blob/812ea7506c4585366176beac39f5e2679de135a3/e2e/hmr/package.json#L11